src/DIRAC/Core/Utilities/Subprocess.py

```diff
@@ -575,7 +575,7 @@ def systemCall(timeout, cmdSeq, callbackFunction=None, env=None, bufferLimit=524
     return result
 
 
-def shellCall(timeout, cmdSeq, callbackFunction=None, env=None, bufferLimit=52428800):
+def shellCall(timeout, cmdSeq, callbackFunction=None, env=None, bufferLimit=20971520):
```

Comment:
This number is clearly smaller than before

Reply:
https://github.com/DIRACGrid/DIRAC/commit/a9e9205af26ee19aea781bbdbe587fa92b4a70e6

Reply:
Oh, so now I should even review my own PRs? 😠 

😄 OK upping to 200 MB

Reply:
`git add --patch` is the first line of review :exclamation: :smiling_face_with_tear: